index.js

```diff
@@ -438,9 +448,16 @@ if (config.http_api_key) {
 app.use('/wallet', walletRouter);
 
 app.listen(config.http_port, config.http_bind_address, () => {
-  console.log(`Hathor Wallet listening on ${config.http_bind_address}:${config.http_port}...`);
+  console.log(`Listening on ${config.http_bind_address}:${config.http_port}...`);
```

Comment:
I am thinking about logging more about the config. The gap limit is being logged when we set it but we could log the `tokenUid` and `connectionTimeout`. What do you think?

Reply:
We could probably do this structured, like `console.log(`Listening on ${config.http_bind_address}:${config.http_port}...`, config);` would log the whole config, but maybe sanitize sensitive info from config if there is any.

Reply:
I prefer to put these piece of information in a separate log. I'll do it and you can review again.

Reply:
Done!

Reply:
Looks great!